theme/theme.ts

```diff
@@ -1,11 +1,719 @@
 /* eslint-disable max-len */
 
-// 🚨 Please update theme in purgableCSSGenerator as well when updating theme here, cannot import it
-// there because of node only supporting cjs modules issue.
 export type Theme = 'blue' | 'teal' | 'green' | 'indigo' | 'orange' | 'purple' | 'gray';
 
 export const THEMES: Theme[] = ['blue', 'green', 'teal', 'indigo', 'orange', 'purple', 'gray'];
 
+export const mapFocusBorder100 = {
```

Comment:
@hussamkhatib Creating a mapping like this for every variant with every theme is not a very scalable and ideal solution, because not only it would be harder to maintain and keep track of as new themes and styles get added or removed, we would also need to import these constants everywhere we want to use theme colors.

I know the new tailwind JIT compiler might be causing issues, so here are a couple of solutions that I can think of -

- Keep the purgable script and remove all these changes you made, just run the script in conjunction with existing `dev` script, with watching for any file changes, this way the script would automatically generate new classnames for the JIT compiler in development mode as well. 
- Look into any other ways of handling theming with tailwind (like [this](https://css-tricks.com/color-theming-with-css-custom-properties-and-tailwind/) maybe). This solution would be more complicated as it might require some major rewrite of existing theming setup, so I would recommend you to try the first solution

Reply:
Reverted back , can you review the new changes.

Reply:
Can you guys add these instructions in readme as well. I wasted a lot of time thinking the purgeable script will not work in v3. It will save a lot of time for newcomers.

Reply:
> Can you guys add these instructions in readme as well. I wasted a lot of time thinking the purgeable script will not work in v3. It will save a lot of time for newcomers.

good point, @hussamkhatib 💯

we will add this to our contribution guide.

Also, if you think you're facing an issue in understanding something, or like want to discuss an approach, request you to raise it first before implementation.
We don't want contributors to revert/add changes to & fro :))

Making a good contribution guide, that's on us. We'll try to make onboarding as easy as possible. 🙌🏻